modules/lang-painless/src/test/resources/rest-api-spec/test/painless/16_update2.yaml

```diff
@@ -28,6 +34,8 @@
   - is_false: script
 
   - do:
+     warnings:
+       - 'specifying lang [painless] as part of the url path is deprecated'
```

Comment:
Also I think we should update the yaml specs to just describe the id and update the yaml tests accordingly.
So that the new behavior gets tested too.

We can add `get_script_deprecated.json`, `delete_script_deprecated.json` etc. to test that the old behaviour and that warnings get returned in the response.